project/src/main/java/com/google/sps/data/ProductItem.java

```diff
@@ -0,0 +1,15 @@
+package com.google.sps.data;
+
+
+
+public class ProductItem{
+    String productId;
+    String productName;
+    String productCategory;
+
+    public ProductItem(String productId, String ProductName, String ProductCategory){
```

Comment:
I'm just wondering, shouldn't a product also contain the product set name or id? Or do you not need it. I know my code has this already so when we are merging we can merge it appropriately.

Reply:
It seems like you may at some point want to be able to link back to a product set from the product itself.  Even if you don't have a use for it right away, it's much more difficult to go back and add this data retroactively than it is to just collect it from the start.  Unless there is some known limitation to getting the product set it when creating the ProductItem, I would recommend that you add it as a member of this class.